packages/peregrine/lib/store/actions/user/asyncActions.js

```diff
@@ -23,6 +25,8 @@ export const signOut = (payload = {}) =>
         await dispatch(clearToken());
         await dispatch(actions.reset());
         await clearCheckoutDataFromStorage();
+        await clearCartDataFromCache(apolloClient);
+        await clearCustomerDataFromCache(apolloClient);
```

Comment:
Here, we clear the rest of the cache data.

Reply:
Beautiful - plan on applying this to the other actions like [cart](https://github.com/magento/pwa-studio/blob/develop/packages/peregrine/lib/store/actions/cart/asyncActions.js#L373)?

Reply:
It looks like maybe we have to add it to the `signIn` async action as well in order to remove it from other handlers.